FAIRsharing/fairsharing.github.io
# Fairsharing clone test cases fix 2147

https://github.com/FAIRsharing/fairsharing.github.io/issues/2147

- Updating modules in package.json
- updating test cases

Fixing ```npm run serve``` errors and warnings.
Fixing ```npm i ``` errors

To test, perform following steps
Please check node version. It should be v16.19.0 and above

- rm -rf node_modules/
- rm package-lock.json
- npm cache clear -f
- npm install
- npm run test:unit
- npm run serve

Please the if all functionalities are working as expected
